model/host/host.go

```diff
@@ -168,6 +169,30 @@ func (h *Host) SetInitializing() error {
 	)
 }
 
+func (h *Host) IncProvisionAttempts() error {
```

Comment:
Write a test.

Reply:
To assert that find and modify works? We can't mock out the DB, so we're just duplicating a mongodb integration test...

Reply:
Ok.